.genx/templates/react/route.hbs

```diff
@@ -22,7 +23,7 @@ const {{pascalCase route.name}} = () => {
   return (
     <section className="{{kebabCase route.name}}-page">
       {{#if route.tiles}}
-      <rapid-layout auto-save-key="{{route.layoutKey}}">
+     <rapid-layout auto-save-key="{{#if environment.REACT_APP_PRODUCTION}}{{route.layoutKey}}{{else}}{{null}}{{/if}}">
```

Comment:
Fyi, this doesn't look right. Handlebars conditional will be processed once - when an app generated, while the dev/prod conditional logic needs to still be there after the app has been generated. Web component version for comparison: https://github.com/genesiscommunitysuccess/blank-app-seed/blob/main/.genx/templates/web-components/route.template.hbs#L14

Reply:
Ah ok, yah I initially had it as a simple conditional statement within the double curly braces, but had a fail when testing. Will check again